include/swift/AST/DiagnosticsSema.def

```diff
@@ -3238,7 +3238,7 @@ ERROR(objc_name_deinit,none,
       "'@objc' deinitializer cannot have a name", ())
 ERROR(objc_name_func_mismatch,none,
       "'@objc' %select{initializer|method}0 name provides "
-      "%select{one argument name|names for %1 arguments}2, but "
+      "%select{one argument label|names for %1 arguments}2, but "
```

Comment:
Ah, in this case it's talking about pieces of an Objective-C selector rather than Swift argument labels. I'm not sure if either "name" or "label" is really appropriate, but for now let's just leave this one as is.

Reply:
Thanks for reviewing, Jordan! I'll revert this one to name.